src/Atom.jl

```diff
@@ -25,7 +25,7 @@ function __init__()
               f = Base.find_source_file(path)
               f !== nothing && (path = f)
           end
-          $(msg)("openFile", f, line-1)
+          $(msg)("openFile", Base.abspath(path), line-1)
```

Comment:
Did you check that this works fine with remote files?

Reply:
This is what I should have mentioned in the PR description.

Actually I don't have an easy setup to use remote servers in Juno right now, and didn't check that, sorry.

Maybe could you check this instead ? We may need to use utilities within utils.jl for this to work perfect in every case I suspect.

Reply:
Works as expected, afaict.

Reply:
Thanks for testing ;)